examples/ola-recorder.cpp

```diff
@@ -60,7 +60,7 @@ DEFINE_s_uint32(iterations, i, 1,
 DEFINE_uint32(start, 0,
               "Time (milliseconds) in show file to start playback from.");
 DEFINE_uint32(stop, 0,
-              "Time (milliseconds) in show file to stop playback at.  If"
+              "Time (milliseconds) in show file to stop playback at. If"
               " the show file is shorter, this option is ignored.");
```

Comment:
Nit, I normally try and put the space at the end of the line so you don't have to remember to keep it in place, like so:
```suggestion
              "Time (milliseconds) in show file to stop playback at. If "
              "the show file is shorter, this option is ignored.");
```